src/haddock/gear/yaml2cfg.py

```diff
@@ -97,69 +126,80 @@ def _yaml2cfg_text(
     exp_level_idx = exp_levels[explevel]
 
     # define set of undesired parameter keys
-    undesired = ("default", "explevel", "type")
+    undesired = ["default", "explevel", "type"]
```

Comment:
I was wondering if we want to still display the `group` in this `haddock3-cfg` output.
If not, we can add it here to remove it
```suggestion
    undesired = ["default", "explevel", "type", "group"]
```